web/modules/custom/joinup_communities/tallinn/src/Plugin/Field/FieldWidget/TallinnEntryWidget.php

```diff
@@ -35,7 +35,12 @@ class TallinnEntryWidget extends WidgetBase {
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $item = $items[$delta];
 
-    $element['#type'] = 'fieldset';
+    $wrapper_classes = 'js-form-wrapper form-wrapper';
+    $element = [
+      '#type' => 'details',
```

Comment:
Yes indeed, still our theme does not have proper styling for fieldset so collapsing and collapsible properties do not work.
Thus, I used details since it is html5 and basic functionality is supported already.